nengo_gui/password.py

```diff
@@ -27,4 +31,4 @@ def prompt_pw():
         p1 = getpass("Enter password: ")
         if p0 == p1:
             return p0
-        print("Passwords do not match. Please try again.")
```

Comment:
This is also code with direct user interaction via stdin and stdout, thus it should stay a `print`.